OSRMTextInstructions.xcodeproj/project.pbxproj

```diff
@@ -761,7 +761,7 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 1;
```

Comment:
Can we just bump the `s.version` and version in bundle? Do we need to bump these `CURRENT_PROJECT_VERSION`?